README.md

```diff
@@ -50,6 +50,32 @@ You can also build just one of the binaries in this repo by passing a name to th
 $ make build BINARIES=amtool
 ```
 
+## Docker
+
+<!--- example of complete docker run command to run a container --->
+<!--- example of a docker-compose.yml service snippet --->
+
+### Ports
+
+* `9093` : main api port
+
+### Volumes
+
+* `/alertmanager`: storage directory
+
+These paths can also be mapped to volumes:
```

Comment:
I am a bit confused by this comment as `/alertmanager` can also be mapped to volumes.

Reply:
`/alertmanager` is declared as a volume in the Dockerfile, so will always be a volume. But any file or directory can be mounted as a volume. I thought a good idea to mention files and directories that people likely also want to mount as volumes. 

I guess the wording can be changed to something like 
```suggestion
You might also want to use volumes for these paths:
```